mkdocs/Configuration.md

```diff
@@ -196,10 +196,163 @@ Requires the [`bukkit.command.credits`](../Permissions#bukkitcommandcredits) per
 	
 ### seed
```

Comment:
```suggestion
### seed
Check out https://minecraft.fandom.com/wiki/Configured_feature for more information on the options below
```
Would be a good idea to add the link to the minecraft wiki at the base of the `seed` header for the users who end up going through these options.